cdap-examples/WikipediaPipeline/src/main/scala/co/cask/cdap/examples/wikipedia/ScalaSparkKMeans.scala

```diff
@@ -35,9 +35,12 @@ class ScalaSparkKMeans extends SparkMain {
     val maxIterations = arguments.get("max.iterations").map(_.toInt).getOrElse(10)
     val k = arguments.get("num.topics").map(_.toInt).getOrElse(10)
 
+    val dataNamespace = sec.getRuntimeArguments.get(WikipediaPipelineApp.NAMESPACE_ARG)
```

Comment:
And what if its not provided ? Should be getting current namespace like all other changes.